packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx

```diff
@@ -208,8 +211,8 @@ export function ContextExpression(contextExpression: ContextExpressionDefinition
       <ContextResultExpressionCell
         key={"context-result-expression"}
         contextExpression={contextExpression}
-        rowIndex={0}
-        columnIndex={0}
+        rowIndex={1}
```

Comment:
I think `rowIndex` should be equal to `rows.length` or something.

Reply:
@tiagobento Changes applied!